ui/pages/Dashboard/components/ProjectsTable.jsx

```diff
@@ -4,9 +4,10 @@ import styled from 'styled-components'
 import Timeago from 'timeago.js'
```

Comment:
okay so I can't comment on the file directly, but why did package-lock change?

Reply:
Did it change? Maybe it is because I tried some front-end libraries.

Reply:
maybe, but since you didn't actually end up using any new front end libraries you should revert this back. You can just checkout the file from dev to reset it